### PR TITLE
Changed from currentDensity to density on EXPERIMENTAL_Table

### PR DESCRIPTION
**Pull request by matheusps**
#### What is the purpose of this pull request?

📢This PR is important to upcoming PR's, that will allow the table to compose it's sections.

🚫Changed from `currentDensity` to `density`. (breaking change)
🚫Deprecate `density` prop of `ButtonDensity`. (breaking change, but will not throw an error on the client)
➕Passed measures through context. 

#### What problem is this solving?

1️⃣Just using `density` is easier to use and makes more sense.
2️⃣ButtonDensity must be easier to use.

#### Who will be affected? 😱

❎Clients using the `ButtonDensity` must remove the `density` prop (not immediate)
☢️Clients comparing the `currentDensity` inside of the `cellRenderer` will stop working or possibly break. So, PR's must be merged together.

#### How should this be manually tested?

`yarn && yarn start`

#### Screenshots or example usage

No need.

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,11 +7,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- From `currentDensity` to `density` on `EXPERIMENTAL_Table`
+
 ## [9.112.15] - 2020-03-10
 
 ### Fixed
 
-- Add `useCallback` in the `showAllColumns` and `hideAllColumns` in the `EXPERIMENTAL_useTableVisibility`.
+- Add `useCallback` in the `sh
+
+wAllColumns`and`hideAllColumns`in the`EXPERIMENTAL_useTableVisibility`.
```
Comment (emersonlaurentino) [inline CHANGELOG.md]:
```suggestion
- Add `useCallback` in the `showAllColumns` and `hideAllColumns` in the `EXPERIMENTAL_useTableVisibility`.
```

Comment (arthurepc) [inline CHANGELOG.md]:
```suggestion
- Add `useCallback` in the `showAllColumns` and `hideAllColumns`in the`EXPERIMENTAL_useTableVisibility`.
```
I don't know if it was github or it was really kind of broken

Comment (matheusps) [reply CHANGELOG.md]: Fixed

Comment (matheusps) [reply CHANGELOG.md]: Fixed

**Pull request merged**